app/core/static/js/candidates.js

```diff
@@ -2,15 +2,145 @@ jQuery(function($) {
   'use strict'
 
   $('.add-popover').each(function() {
-    $(this).popover({container:'body', html:true})
+    $(this).popover({ container: 'body', html: true })
   })
+})
 
-  $('.statistic').each(function(){
-    var id = this.id.replace('statistic-', '')
+function qpsToString(qps) {
+  return Object.keys(qps)
+    .reduce(function(arr, key) {
+      if (qps[key]) {
+        arr.push(key + '=' + qps[key])
+      }
 
-    $.getJSON($(this).data('url'), function(data) {
-      var margin = data.summary.titles.length > 1 ? 0 : 30
-      $('#statistic-' + id).highcharts({
+      return arr
+    }, [])
+    .join('&')
+}
+
+function getCurrentURLWithoutPage() {
+  return location.href.replace(/&?page=\d+/, '')
+}
+
+Vue.component('candidate-pagination', {
+  data: function() {
+    return {
+      url: getCurrentURLWithoutPage()
+    }
+  },
+  computed: {
+    baseUrl: function() {
+      if (this.url.search(/\?/) === -1) {
+        return this.url + '?'
+      }
+
+      return this.url + '&'
+    },
+    prevPage: function() {
+      return this.page === 1 ? 1 : this.page - 1
+    },
+    nextPage: function() {
+      return this.page < this.pages.length ? this.page + 1 : this.page
+    },
+    prevDisabled: function() {
+      return this.prevPage === this.page
+    },
+    nextDisabled: function() {
+      return this.nextPage === this.page
+    }
+  },
+  template: `
+    <nav class="text-center">
+      <ul v-if="pages.length" class="pagination">
+        <candidate-pagination-page :disabled="prevDisabled" content="«" :page="prevPage"></candidate-pagination-page>
+        <candidate-pagination-page
+          v-for="i in pages"
+          :current="page"
+          :content="i"
+          :page="i"
+          :key="i"
+        >
+        </candidate-pagination-page>
+        <candidate-pagination-page :disabled="nextDisabled" content="»" :page="nextPage"></candidate-pagination-page>
+      </ul>
+    </nav>
+  `,
+  props: ['page', 'pages']
+})
+
+Vue.component('candidate-pagination-page', {
+  computed: {
+    base: function() {
+      let url = getCurrentURLWithoutPage()
+
+      if (url.search(/\?/) === -1) {
+        return url + '?'
+      }
+
+      return url + '&'
```

Comment:
Might be worth pulling this into something like `getBaseUrl(url)` since you do the same thing above? (Maybe you find a better name for the function that appends `?` or `&`)

Reply:
Or you refactor the `url` function below:
```
return joinUrl(getCurrentURLWithoutPage(), 'page=' + this.page)
```